.travis.yml

```diff
@@ -31,3 +32,10 @@ script:
   - go vet ./management/...
   - test -z "$(golint ./Gododir/... | tee /dev/stderr)"
   - go vet ./Gododir/...
+  - errcheck -ignoretests ./arm/... 
```

Comment:
can be shortened: `set -e; for v in ./arm... ./datalake-store/... ./Gododir/...; do errcheck -ignoretests "$v"; done`